app/Http/Requests/Three/PostRequest.php

```diff
@@ -27,11 +27,14 @@ public function rules()
             'campaign_id' => 'required',
             'campaign_run_id' => 'required_with:campaign_id|integer',
             'northstar_id' => 'nullable|objectid',
+            'type' => 'required|string',
```

Comment:
Do we want to validate what types are set here? `in:photo,voterreg`

Reply:
@DFurnes I was wondering that as well... I can add it! We will just have to keep an eye on new types coming in and update this validation accordingly.